cmd/node/main.go

```diff
@@ -199,7 +199,7 @@ VERSION:
 	}
 
 	//useTermui
-	useTermui = cli.BoolFlag{
+	useTermui = cli.BoolTFlag{
```

Comment:
Typo
Should change the logic as the app should always start in termui interface. Maybe the bool should be called "use-log-view" and the condition on line 500 should be negated.